akshatshukla13.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "Akshat Shukla",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "As An enthusiastic tech aficionado, I am constantly fueled by my passion to explore and discover",
+  "avatar": "https://media.licdn.com/dms/image/D5603AQFA-E-9DFGSUA/profile-displayphoto-shrink_800_800/0/1668493033974?e=1684368000&v=beta&t=RRQZUtcrRu3hXT6yIZoGZ65ydkLUFVakMK4C43lWbPE",
```

As of our recent update, we no longer need the `displayStatsPublic` & `avatar` keys. So please remove these fields from your json. 

```suggestion
  "type": "personal",
  "bio": "As An enthusiastic tech aficionado, I am constantly fueled by my passion to explore and discover",
```